### PR TITLE
Pass the command line options to simulation server

### LINKED ISSUE #7170: Forwarding command line options to simulation server
### Description
The command line options specified in OMEdit are not forwarded to the simulation server. Do we want to split it up into options for modelling and another one for simulation? I think that would be reasonable - but maybe slightly confusing. But those options are only meant to be used by experts anyways.

### Screenshots
image

### PR DESCRIPTION
**Pull request by adeas31**
Fixes #7170

### CODE REVIEW

File: OMEdit/OMEditLIB/OMS/OMSSimulationOutputWidget.cpp [hunk 1]
```diff
@@ -126,6 +126,10 @@ OMSSimulationOutputWidget::OMSSimulationOutputWidget(const QString &cref, const
     QStringList args(QString("%1/share/OMSimulator/scripts/OMSimulatorServer.py").arg(Helper::OpenModelicaHome));
     args << QString("--endpoint-pub=%1").arg(QString(endPoint));
     args << QString("--model=%1").arg(fileName);
+    OMSimulatorPage *pOMSimulatorPage = OptionsDialog::instance()->getOMSimulatorPage();
+    int logLevel = pOMSimulatorPage->getLoggingLevelComboBox()->itemData(pOMSimulatorPage->getLoggingLevelComboBox()->currentIndex()).toInt();
+    args << QString("--logLevel=%1").arg(logLevel);
+    args << QString("--option=%1").arg(pOMSimulatorPage->getCommandLineOptionsTextBox()->text());
```
Comment (lochel) [inline OMEdit/OMEditLIB/OMS/OMSSimulationOutputWidget.cpp:132]: Do we might get escaping issues?

Comment (adeas31) [reply OMEdit/OMEditLIB/OMS/OMSSimulationOutputWidget.cpp:132]: Maybe if we have any string based options but I don't see any in the Flags.cpp

Comment (lochel) [reply OMEdit/OMEditLIB/OMS/OMSSimulationOutputWidget.cpp:132]: There are a few, e.g. result file, temp directory. But they are not really intended to be used together with OMEdit.

Comment (adeas31) [reply OMEdit/OMEditLIB/OMS/OMSSimulationOutputWidget.cpp:132]: As long as the user quotes the options we are fine.

**Review by lochel — APPROVED** [review]

**Comment by lochel** [comment]
I'll merge it now because otherwise I cannot simulate my models. Thanks for the quick fix!

**Pull request merged**